pandas/tools/plotting.py

```diff
@@ -2262,11 +2262,13 @@ def result(self):
 
 def _plot(data, x=None, y=None, subplots=False,
           ax=None, kind='line', **kwds):
+
+    invalid_ct_fmtstr = "'%s' is not a valid chart type"
```

Comment:
How would you formulate the message? `'blabla' is not a valid value for the keyword argument 'kind'`?

Is it possible to replace `chart type` by `chart kind`? I am not a native speaker - to me it somehow sounds strange.

at this point `kind` can only be a string. It gets set in line 2265:  `kind = _get_standard_kind(kind.lower().strip())`